testsV2/ut_repyv2api_safedictests.r2py

```diff
@@ -0,0 +1,179 @@
+"""
+Test interactions with SafeDict.
```

Comment:
Would you mind listing all the interactions, i.e. `SafeDict(UserDict.DictMixin)` methods here?

Reply:
I'll list everything I test, which is mostly congruent with what `dict` allows, save for the (outcommented) creation methods from kwargs etc.

The `SafeDict` methods aren't quite the same thing, and we don't / can't (in Repy) access the double-underscore methods. `in`, `del` etc. are tested implicitly.